cnf-tests/TESTLIST.md

```diff
@@ -134,7 +134,12 @@ The cnf tests instrument each different feature required by CNF. Following, a de
 
 | Test Name | Description |
 | -- | ----------- |
+| [performance]  Should have kernel param rcutree.kthread | XXXXXXXX | 
```

Comment:
these XXXXXX (there's few more below) don't look right, please check

Reply:
This generated by default when I'm running the relevant make target.
Changing the XXX cause the CI to fail

Reply:
@SchSeba any insights here?

Reply:
it's because the tests are missing a description 

AFAIK, just fill in the description in the json file.
and you'll also get the new tests documented :-)

Reply:
that is right you need to update the json here for example
https://github.com/openshift-kni/cnf-features-deploy/pull/1357/files#diff-cfad2c15f9822954c50b9f3f08ce93166ba405eb510950e86e4c045a4eb22fa9R37
and generate the docs

Reply:
Maybe I did it the other way around and updated the docs themselves instead of the json. I'll give it a try